src/windows/system.rs

```diff
@@ -415,11 +423,23 @@ impl SystemExt for System {
     }
 
     fn get_os_version(&self) -> Option<String> {
-        get_reg_string_value(
+        let major = get_reg_value_u32(
             HKEY_LOCAL_MACHINE,
             "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
-            "ProductName",
-        )
+            "CurrentMajorVersionNumber",
+        );
+
+        let minor = get_reg_value_u32(
+            HKEY_LOCAL_MACHINE,
+            "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
+            "CurrentMinorVersionNumber",
+        );
+
+        Some(format!(
+            "{}.{}",
```

Comment:
This function returns what is is supposed to return but i'm not sure if that's the best value to return here.  
So the minor version is only used here to know if you are using windows 8 or 8.1 and if you ask me in windows land v8.1 to 8 has a different meaning than the MacOS minor versions.  

Looking at the [win 10 version history](https://en.wikipedia.org/wiki/Windows_10_version_history) it also shows a windows 10 build version and i think that would be the perfect value if only windows didn't decide to put letters in there *(`20H2` WHY WINDOWS WHYYY!!)*??   

So as windows 10 is now at least 5 and seem to have ditched the major and minor versions it might be better to return the build number here as you can sort of get the mojor *(as in win 10 and the eara of windows 10 like `Threshold x` or `Redstone x`)* and you can actually get a sort of minor version from it that's kinda usefull.

Reply:
I updated this call to move to `"{} ({})", CurrentMajorVersionNumber, CurrentBuildNumber`, how do you like this? I think it would give any one a good way to know what version of Windows and good knowledge of about how outdated their system is 😅

Reply:
Looks good